ibis-server/app/main.py

```diff
@@ -23,20 +49,22 @@ def health():
 
 
 @app.get("/config")
-def config():
+def provide_config():
     return get_config()
 
 
-@app.exception_handler(QueryDryRunError)
-async def query_dry_run_error_handler(request, exc: QueryDryRunError):
-    return PlainTextResponse(str(exc), status_code=422)
+@app.patch("/config")
+def update_config(config_model: ConfigModel):
```

Comment:
I saw you `get_config().init_logger` at L17. Should we init the logger again after updating the config?

Reply:
We will remove the old logger and add a new logger when the config is updated in the `config.py`.